src/wwwroot/services/templates.js

```diff
@@ -80,17 +81,17 @@
       });
     }
 
-    function save(templateModel) {
+    function save(templateModel, isLegacyTemplate) {
```

Comment:
In general boolean parameters are not a good idea, what do you think about splitting _save_ in the same way that you did with _getTemplateWithBody_?

Reply:
Ok I will check this method to split it 👍